{
    "crime": "3841. Ljava/io/IOException;getCause->Ljava/lang/AssertionError;getCause",
    "permission": [],
    "api": [
        {
            "class": "Ljava/io/IOException;",
            "method": "getCause",
            "descriptor": "()Ljava/lang/Throwable;"
        },
        {
            "class": "Ljava/lang/AssertionError;",
            "method": "getCause",
            "descriptor": "()Ljava/lang/Throwable;"
        }
    ],
    "score": 1
}